README.md

```diff
@@ -91,78 +44,42 @@ See "Reference architecture" in Authoring Guidelines in the public documentation
 https://terraform-ibm-modules.github.io/documentation/#/implementation-guidelines?id=reference-architecture
 -->
 
-## Usage
-
-<!--
-Add an example of the use of the module in the following code block.
-
-Use real values instead of "var.<var_name>" or other placeholder values
-unless real values don't help users know what to change.
--->
-
-```hcl
-
-```
+<!-- BEGIN OVERVIEW HOOK -->
```

Comment:
What's the difference between the summary and overview sections?  

In the overview - why have a link back to the same module (1st link below).  

I suggest reviewing the README format from the module template and adhering to that as well as looking at other mature modules as examples.

Reply:
Removed this section.